### ISSUE TITLE
RFC 8267 states it applies ONLY to RPC/RDMA v1

### ISSUE THREAD

**Issue by chucklever** [chucklever/i-d-rpcrdma-version-two#1]
Dave Noveck points out that RFC 8267 has a paragraph that states that the ULB specified there applies only for RPC/RDMA v1. In his opinion this is not a valid restriction, but in any event we need some language somewhere that corrects this for v2. Suggest that the v2 specification make the correction.

**Comment by dnoveck** [comment]
One way to deal with this is to have the V2 spec update rfc 8267.  If the IESG is not satisfied with that you may have to do an rfc8267bis☹️.

If you add send chaining, additional work would be needed.  Will discuss elsewhere.

**Comment by chucklever** [comment]
I'm not sure I completely understand the issue here. Can you spell it out here in the comments?

For example, what's the problem with writing a separate document that is the ULB for NFS on RPC/RDMA v2? Such a document can refer to RFC 8267 pretty extensively, and provide modified text for the few areas that will be different with RPC/RDMA v2. That approach would be the correct one, IMO.

**Comment by dnoveck** [comment]
*   For example, what's the problem with writing a separate document that is the ULB for NFS on RPC/RDMA v2?

I’m not saying it’s a big problem.   I just saying you have to write, review it, bring it through the IESG, etc.

  *   Such a document can refer to RFC 8267 pretty extensively, and provide modified text for the few areas that will be different with RPC/RDMA v2.

I don’t think the IESG is going to like that sort of cut-and-paste approach.  You might get by, but if you go ahead with something that they don’t like, the process gets really annoying.

  *   That approach would be the correct one, IMO.
The problem is a that the IESG has a veto.

**Comment by chucklever** [comment]
>> Such a document can refer to RFC 8267 pretty extensively, and provide modified text for the few areas that will be different with RPC/RDMA v2.

> I don’t think the IESG is going to like that sort of cut-and-paste approach. You might get by, but if you go ahead with something that they don’t like, the process gets really annoying.

Fair enough, but building a whole new document instead of a document full of pointers should not be much more work, since RFC 82678 has already been done.
